### ISSUE TITLE
Nvda is not reading while selecting a text in Skype for Business

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#10758]
### Steps to reproduce:
Start typing a new message.
Type some word.
press ctrl + shift + left arrow.
You can also try to select characters with shift + arrows on the right or left.

### Actual behavior:
nvda does not announce the text being selected
### Expected behavior:
nvda must announce the text being selected.
Everything works fine in word and other text editors.
it is possible to confirm that the text has been selected with nvda + shift + s.
#### NVDA installed/portable/running from source:
portable
#### NVDA version:
alfa 19604
#### Windows version:
10 1909
#### Name and version of other software in use when reproducing the issue:
: lync.exe, Microsoft Office 2013 15.0.5159.1000

#### Does the issue still occur after restarting your PC?
yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
no

**Comment by Adriani90** [comment]
This is not reproducible in NVDA 2019.3.1 with Skype@ for Business 2016 (16.0.4978.1000)
@fernando-jose-silva are you still having this issue?

**Comment by fernando-jose-silva** [comment]
I temporarily lost access to this problem machine, this machine is at my place of work, where we are away due to a pandemic.
  The notebook we received to work at home, has a version of Skype for Business 2016, and I don't have this problem.
  Just regain access to the troubled machine after the quarantine has ended.

Enviado do meu iPhone

**Comment by Adriani90** [comment]
@fernando-jose-silva thanks for reporting. I think this issue is not reproducible in newest NVDA version  and newest skype for business version. But feel free to comment on it and we can reopen if this is still a problem for you.
For now I am closing this as works for me.
